lib/logstash/outputs/file.rb

```diff
@@ -58,6 +58,18 @@ class LogStash::Outputs::File < LogStash::Outputs::Base
   # in such a file, the plugin will created a gain this file. Default => true
   config :create_if_deleted, :validate => :boolean, :default => true
 
+  # Dir access mode to use. Note that due to the bug in jruby system umask
+  # is ignored on linux: https://github.com/jruby/jruby/issues/3426
+  # Setting it to -1 uses default OS value.
+  # Example: "dir_mode" => 0750
```

Comment:
Please encapsulate the example in back-ticks, e.g.

```
# Example: `"dir_mode" => 0750`
```

As this will format it like this in the rendered documentation:

> Example: `"dir_mode" => 0750`